test/extended/templates/templateservicebroker_e2e.go

```diff
@@ -72,6 +75,12 @@ var _ = g.Describe("[templates] templateservicebroker end-to-end test", func() {
 		})
 		o.Expect(err).NotTo(o.HaveOccurred())
 
+		//TODO - when https://github.com/kubernetes-incubator/service-catalog/pull/939 sufficiently progresses, we should see about removing
+		// this add-role-to-user, as it should not be required
+		// give test user access to the openshift project's templates ... now needed with provision's SAR for get on templates (previously commented out)
+		err = cli.AsAdmin().Run("policy").Args("add-role-to-user", "view", cli.Username(), "-n", "openshift").Execute()
```

Comment:
now this one i agree should not be needed because once we're respecting the user's groups, the group permissions should be enough to give the user access to things in the openshift namespace.

but i don't see why it's reliant on the SC changes...  we're using our own broker client here which provides a proper user object (and thus should have the proper groups set), right?  so shouldn't this already be working?